src/blocks/logo/scss/_editor.scss

```diff
@@ -1,5 +1,7 @@
-.wp-block[data-type="novablocks/logo"] {
-    a {
-        pointer-events: none;
-    }
-}
\ No newline at end of file
+.site-branding {
+    pointer-events: none;
+}
+
+.c-logo__inverted {
+    display: none;
+}
```

Comment:
`.c-logo__inverted` nu mai e folosit in markupul din nova blocks deci nu mai e nevoie de bucatica asta.

Reply:
Fixed.